crates/steel-core/src/steel_vm/builtin.rs

```diff
@@ -324,7 +324,15 @@ impl BuiltInModuleRepr {
 
     // This _will_ panic given an incorrect value. This will be tied together by macros only allowing legal entries
     pub fn get(&self, name: String) -> SteelVal {
-        self.values.get(name.as_str()).unwrap().clone()
+        match self.values.get(name.as_str()) {
+            Some(v) => v.clone(),
+            None => {
+                panic!(
+                    "symbol {name} not found in module {module_name}",
```

Comment:
Does it make sense to also add a "did you mean...".

If so, is there already some existing reference to this functionality in the codebase I can take a look at?

Reply:
Related: The `clap` crate uses version `0.10` of `strsim`. `normalized_levenshtein` seems like a good way to find the best candidate

https://docs.rs/strsim/latest/strsim/fn.normalized_levenshtein.html

Reply:
Done, used the `strsim` crate to find closest match and wrote some unit tests to demonstrate the range.